### PR TITLE
Fix: Repo name has changed, fix readme accordingly.

### LINKED ISSUE #40: Repo name has changed, fix readme accordingly
Repo name for this project has changed https://github.com/code-collabo/collabo-community-app, this has not yet reflected in the repo's readme yet.

Fix the readme accordingly.

### PR DESCRIPTION
**Pull request by Dkingofcode**
**This pull request makes the following changes**
* Fix https://github.com/code-collabo/docs/issues/40

**General checklist**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**

Replace this text with the testing checklist from the issue this pull request fixes.

Ping @code-collabo/collabo-community

### CODE REVIEW

**Comment by Dkingofcode** [comment]
@Ifycode, submitted pull request, awaiting review.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# Collabo Web App
+# collabo-community-app
```
Comment (Ifycode) [inline README.md]: **_Review comment 1:_** Please use idea from the previous title @Dkingofcode 
There are no hyphens and each word starts with a capital letter.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://code-collabo.gitbook.io/collabo-contributor/collabo-web-app-project/contribute-to-collabo-web-app) [![GitHub pull requests](https://img.shields.io/github/issues-pr/code-collabo/collabo-web-app?color=goldenrod)](https://github.com/code-collabo/collabo-web-app/pulls)
 
-The Collabo Web App is an internal project aimed at promoting the Code Collabo Community, its projects, and programs. It also serves as a showcase for our web development tools like node-mongo and Collabo CSS framework. Opportunities for UI/UX and graphic design contributions are also available.
+The Collabo community App is an internal project aimed at promoting the Code Collabo Community, its projects, and programs. It also serves as a showcase for our web development tools like node-mongo and Collabo CSS framework. Opportunities for UI/UX and graphic design contributions are also available.
```
Comment (Ifycode) [inline README.md]: **_Review comment 2:_** "The Collabo community App" -  Make the word "community" begin with a capital letter similar to what was there before @Dkingofcode

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Dkingofcode) [reply README.md]: Will endeavour to make those changes appropriately.

**Comment by Ifycode** [comment]
Make the changes in this branch and PR @Dkingofcode 
Don't create a new branch.

**Comment by Dkingofcode** [comment]
@Ifycode I have made the changes in this branch.  Awaiting review.

**Review by Ifycode — APPROVED** [review]
Approved

**Pull request merged**